__tests__/app.test.js

```diff
@@ -101,9 +101,21 @@ describe("app", () => {
       expect(article).toHaveProperty("topic", expect.any(String));
       expect(article).toHaveProperty("created_at", expect.any(String));
       expect(article).toHaveProperty("votes", expect.any(Number));
-      expect(article).toHaveProperty("article_img_url", expect.any(String));
+      expect(article).toHaveProperty("article_img_url", expect.any(String))
+      expect(article).toHaveProperty("comment_count", 2);
```

Comment:
Ideally, we shouldn't have had to go back and change this test but instead write a new one. We could have achieved this by using .toMatchObject or .objectContaining here instead of .toHaveProperty